satpy/composites/__init__.py

```diff
@@ -823,7 +823,10 @@ def __call__(self, projectables, nonprojectables=None, **attrs):
         new_attrs.update({key: val
                           for (key, val) in attrs.items()
                           if val is not None})
+        resolution = new_attrs.get('resolution', None)
         new_attrs.update(self.attrs)
+        if resolution is not None:
+            new_attrs['resolution'] = resolution
```

Comment:
I don't understand why this is needed. Shouldn't `new_attrs` already have `resolution`? Or does `self.attrs` have a `resolution` of `None` which is overwriting the `new_attrs['resolution']`?

Reply:
Yes exactly, self.attrs has calibration, polarisation, modifiers, resolution ,level set to None